### ISSUE TITLE
Returns only first 30 videos related to the profile

### ISSUE THREAD

**Issue by leklement** [mouhamadalmounayar/TiktokScraper#8]
In one request Tiktok only returns first 30 videos
There is cursor parameter in request which can help you retrieve all videos in while cycle

**Comment by mouhamadalmounayar** [comment]
Yes, indeed. I opened an issue for it and solved this (Check issue #5 ) . I gave the user the choice to fetch the first 35 videos or use the flag -a to fetch all videos.
